### DISCUSSION TITLE
AsciiDoc syntax highlighting support?

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#10611] [Q&A] [answered]
Does Weblate provide syntax highlighting for AsciiDoc (.adoc) markup in PO files?

I found no issue or mention in documentation. All I found was support for other formats in [version 4.4 as per Changes page](https://docs.weblate.org/pt/weblate-4.4/changes.html?highlight=syntax%20highlight#weblate-4-4)

**Comment by nijel** [comment, marked as answer]
No, there is currently no support for AsciiDoc.
